js/db.ts

```diff
@@ -57,6 +57,7 @@ async function initializeDB(name: string) {
 
   if (urlParam("enableFirestorePersistence")) {
     await firebase.firestore().enablePersistence({ synchronizeTabs: true });
+    await firebase.firestore().disableNetwork();
```

Comment:
This seems like a strange thing be adding in this PR.  I don't remember exactly but I think the cypress tests have another way of disabling the network.  So I think this change should be removed unless it can be justified.